pages/1.12/installing/production/advanced-configuration/configuration-reference/index.md

```diff
@@ -87,6 +87,7 @@ This page contains the configuration parameters for both DC/OS Enterprise and DC
 | [adminrouter_tls_1_1_enabled](#adminrouter-tls-1-1-enabled)    | Indicates whether to enable TLSv1.1 support in Admin Router. |
 | [adminrouter_tls_1_2_enabled](#adminrouter-tls-1-2-enabled)    | Indicates whether to enable TLSv1.2 support in Admin Router. |
 | [adminrouter_tls_cipher_suite](#adminrouter-tls-cipher-suite)    | Override the default TLS cipher suite in Admin Router. |
+| [adminrouter_x_frame_options](#adminrouter-x-frame-options)    | Set the `X-Frame-Options` header value for the DC/OS UI. Default is set to `SAMEORIGIN` |
```

Comment:
We should note that this has been introduced only with 1.12.4.

(CC @jpe442 @DominikDary @mhrabovcin)